ra_aid/llm.py

```
@@ -1,21 +1,23 @@
 import os
-from typing import Any, Dict, Optional
+from typing import Any, Dict, List, Optional
 
-from openai import OpenAI
 from langchain_anthropic import ChatAnthropic
 from langchain_core.language_models import BaseChatModel
+from langchain_core.messages import BaseMessage
```

Ruff (0.8.2)

`langchain_core.messages.BaseMessage` imported but unused

Remove unused import: `langchain_core.messages.BaseMessage`

(F401)